### PR TITLE
add documenation about the api

### PR DESCRIPTION
**Pull request by diegodurs**

### CODE REVIEW

**Comment by gerbal** [comment]
FYI, you can just write these pages as Markdown

**Comment by SimonMiaou** [comment]
Not a really readable PR

File: index.html [hunk 1]
```diff
@@ -1,53 +1,558 @@
...
+  <section class="page-header">
+    <h1 class="project-name">Rights'Up</h1>
+    <h2 class="project-tagline">Developer Guide </h2>
+  </section>
+  <section class="main-content">
+    <h3>
+      <a id="overview" class="anchor" href="#overview" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Welcome to Rights'Up API documentation.
+    </h3>
+    <p>
+      All requests should be made against our Staging API endpoint: <code>http://www.rightsup-staging.com:180</code>. For requests to be authorized, we currently use HTTP header to old an <code>API_TOKEN</code>key which value should be set to <code>b49a0d5f3fbe</code>.
```
Comment (SimonMiaou) [inline index.html]: to hold*

Comment (SimonMiaou) [inline index.html]: A free token for everyone on internet that want to try our staging?

**Pull request merged**